dex/msgjson/types.go

```diff
@@ -1079,6 +1079,7 @@ type FeeAsset struct {
 type ConfigResult struct {
 	CancelMax        float64   `json:"cancelmax"`
 	BroadcastTimeout uint64    `json:"btimeout"`
+	TxWaitExpiration uint64    `json:"txWaitExpiration"`
```

Comment:
Off the top of my head, I don't think there's any reason to communicate this in `ConfigResult` to the client.